Header header
string inputCommand
Monitor[] monitors
Objective[] objectives
Behavior[] behaviors
string execBehavior
